libensemble/tests/functionality_tests/test_mpi_gpu_settings.py

```diff
@@ -147,10 +147,10 @@
 
         # check having only cores_per_node
         if run_set == "jsrun":
-            del libE_specs["platform_specs"].logical_cores_per_node
+            libE_specs["platform_specs"].logical_cores_per_node = None
```

Comment:
I hope this doesn't need to be set. Doesn't it get the default in the next call to libE (or .run())